keps/sig-apps/3939-consider-terminating-pods-jobs/README.md

```diff
@@ -273,30 +289,19 @@ type JobStatus struct {
 
 ### Implementation
```

Comment:
Implementation details might not be that important in a KEP, but what you detail here is high-level enough that you can keep it.

What's more important is semantics, and there are some missing here:
- What happens when a Job is suspended? Ideally, we would keep track of the terminating pods.
- What happens when we hit a backoffLimit or the rule "FailJob" for failure policies. Ideally we keep track of terminating pods here as well. Then we have to decide whether we can add the Failed condition before all pods terminate. We could.